src/plugins/intel_npu/src/plugin/src/plugin.cpp

```diff
@@ -752,7 +756,36 @@ std::shared_ptr<ov::ICompiledModel> Plugin::import_model(std::istream& stream, c
     OV_ITT_SCOPED_TASK(itt::domains::NPUPlugin, "Plugin::import_model");
     OV_ITT_TASK_CHAIN(PLUGIN_IMPORT_MODEL, itt::domains::NPUPlugin, "Plugin::import_model", "merge_configs");
 
-    const std::map<std::string, std::string> propertiesMap = any_copy(properties);
+    // If NPUW is active - import via NPUW
+    auto useNpuwKey = ov::intel_npu::use_npuw.name();
+    ov::AnyMap localProperties = properties;
+    if (localProperties.count(useNpuwKey) && localProperties.at(useNpuwKey).as<bool>() == true) {
+        auto llm_enabled = ov::intel_npu::npuw::llm::enabled.name();
+        // Only dynamic stateful models are supported for now supported
+        if (!localProperties.count(llm_enabled) || localProperties.at(llm_enabled).as<bool>() == false) {
+            OPENVINO_THROW("Cannot import non-dynamic NPUW model!");
+        }
```

Comment:
I think you don't need to check any properties here. Properties which determine the flow should be in the blob.

Reply:
Done